mlflow/utils/validation.py

```diff
@@ -234,12 +232,8 @@ def _validate_param_name(name):
 
 
 def _validate_tag_name(name):
```

Comment:
These validation APIs are used by storage backends (e.g. `validate_param_name` is used by FileStore, `validate_tag_name` is used by FileStore & SqlAlchemyStore via `validate_tag`). The backends should still be validating the names, but the clients should not be. Rather than changing the validation util function implementations, we should be changing where these functions are called (i.e. remove the function calls from the MlflowClient layer).

Reply:
Specifically, we should update https://github.com/mlflow/mlflow/blob/HEAD/mlflow/tracking/_tracking_service/client.py

Reply:
totally read that ticket wrong. Moved the logic to correct location.